korean/ui.py

```diff
@@ -18,8 +18,8 @@
 ui_actions = {}
 
 dictionaries = [
-    ("None", _("None")),
```

Comment:
Does this mean Anki no longer supports localization?

Reply:
Anki moved to [fluent](https://projectfluent.org/) in 2020 and since then gettext is no longer a build dependency so you get deprecation warnings when starting anki. Tbh I'm not that familiar with gettext but I tried to change the anki languages on 0.18 and haven't been able to see any localizations. Not sure where it should even get them from. From what I've seen they don't do anything right now. [Anki themselves](https://translating.ankiweb.net/anki/developers.html#add-ons) don't recommend fluent for addons at this time. Proper localization support would probably involve a bit more.